src/totalOrder.ml

```diff
@@ -1,14 +1,10 @@
-type comparison =
+(** A type represent a total order relationship *)
```

Comment:
Why totalOrder vs. ordering? Not opposed, just curious to the rationale. For reference, I took the name from Haskell: https://hackage.haskell.org/package/base-4.9.1.0/docs/Data-Ord.html